src/secrets.js

```diff
@@ -42,7 +44,14 @@ async function getSecrets(secretRequests, client) {
             } catch (error) {
                 const {response} = error;
                 if (response?.statusCode === 404) {
-                    throw Error(`Unable to retrieve result for "${path}" because it was not found: ${response.body.trim()}`)
+                    notFoundMsg = `Unable to retrieve result for "${path}" because it was not found: ${response.body.trim()}`;
+                    const ignoreNotFound = (core.getInput('ignoreNotFound', { required: false }) || 'false').toLowerCase() != 'false';
+                    if (ignoreNotFound) {
+                        core.error(`✘ ${notFoundMsg}`);
```

Comment:
No strong opinion but I can see this being a warning instead of an error since the input flag tells us a missing secret is acceptable.

Reply:
I thought about this a bit and since it is an error returned by Vault I think I will keep it as an error in the action. Thanks @maxcoulombe !